R/restore.R

```diff
@@ -598,35 +570,38 @@ installPkg <- function(pkgRecord,
   }
 
   # Annotate DESCRIPTION file so we know we installed it
-  annotatePkgDesc(pkgRecord, project, lib)
+  withCallingHandlers(
+    annotatePkgDesc(pkgRecord, project, lib),
+    error = function(e) {
+      unlink(pkgInstallPath, recursive = TRUE)
+      return(e)
```

Comment:
Nit: calling handlers don't alter control flow, so I believe the `return()` usage here is superfluous / ignored.

Reply:
Thanks. Confirmed that this is the case.